modules/core/src/v2/coins/utxo/WalletKeys.ts

```diff
@@ -0,0 +1,96 @@
+/**
+ * Classes for deriving key triples for wallet addresses.
+ *
+ * By default, BitGo wallets consist of a triple of bip32 extend keypairs.
+ * Every wallet address can be identified by _(chain: number, index: number)_.
+ * The key set for a particular address can be obtained by deriving with the path
+ * `0/0/${chain}/${index}`. (In rare cases the prefix 0/0 can be different)
+ *
+ * Since we never use other derivations for utxo address scripts, the classes defined here only
+ * allow exactly one level of derivation.
+ */
+
+/**
+ * @prettier
+ */
+import * as bip32 from 'bip32';
+import { Triple } from '../../triple';
+
+export function eqPublicKey(a: bip32.BIP32Interface, b: bip32.BIP32Interface): boolean {
+  return a.publicKey.equals(b.publicKey);
+}
+
+/**
+ * Base method for
```

Comment:
Looks like this comment is incomplete or unintended?

Reply:
oops

will fix in follow-up